MAiD - Case Management App/force-app/main/default/objects/Case/fields/If_Other_Race_Specify__c.field-meta.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>If_Other_Race_Specify__c</fullName>
+    <encryptionScheme>None</encryptionScheme>
```

Comment:
@RanadheerRG: Please provide description and help text if provided. Or else, please check with Lara. Thanks!

Reply:
Updated the help text